src/components/network/network-map-tab.tsx

```diff
@@ -846,13 +846,26 @@ export const NetworkMapTab = ({
         if (!rootNodeId) {
             return;
         }
+        if (studyUpdatedForce.eventData.headers) {
+            console.log('=========== rootNetworkModified !', studyUpdatedForce.eventData.headers[UPDATE_TYPE_HEADER]);
+            if (
+                studyUpdatedForce.eventData.headers[UPDATE_TYPE_HEADER] === 'rootNetworkModified' &&
+                studyUpdatedForce.eventData.headers['rootNetwork'] === currentRootNetworkUuid
+            ) {
+                setInitialized(false);
+                setIsRootNodeGeoDataLoaded(false);
+                dispatch(resetMapEquipment());
+                return;
+            }
+        }
```

Comment:
It will be cleaner I think if you move this notification consumption code in a dedicated effect